test/integration/hello-test.spec.js

```diff
@@ -14,15 +14,14 @@
 // jshint ignore: start
 /*global describe, it*/
 
-import helloWorld from "../../src/index.js";
-import * as assert from 'assert';
-import * as chai from 'chai';
+const assert = require('assert');
```

Comment:
Disable integration tests by disabling/removing it from package.json cover and test:integ commands.